server/views/layout.njk

```diff
@@ -1,6 +1,25 @@
 {% extends "govuk/template.njk" %}
 
 {% block head %}
+  <!-- Google Tag Manager -->
+      <script nonce='{{ cspNonce }}'>
+        ;(function(w,d,s,l,i) {
+          w[l]=w[l]||[];
+          w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});
+          var f=d.getElementsByTagName(s)[0],
+            j=d.createElement(s),
+            dl=l!='dataLayer'?'&l='+l:'';
+            
+          j.async=true;j.src=
+          'https://www.googletagmanager.com/gtm.js?id='+i+dl '{{ tagManagerEnvironment | safe }}';
```

Comment:
Don't you need to drop `googleTagId` in somewhere?

Reply:
Actually this line probably won't work, have a nosey in your browser console when you run the app locally.

Reply:
Needs to be `'https://www.googletagmanager.com/gtm.js?id='+i+dl+'{{ tagManagerEnvironment | safe }}'` maybe?